projects/observability/src/pages/api-trace-detail/api-trace-detail.service.ts

```diff
@@ -40,7 +40,7 @@ export class ApiTraceDetailService implements OnDestroy {
         startTime: paramMap.get(ApiTraceDetailService.START_TIME_PARAM_NAME) as string | undefined
       })),
       takeUntil(this.destroyed$),
-      shareReplay(1)
+      shareReplay({ bufferSize: 1, refCount: true })
```

Comment:
not needed, because we unsubscribe on component destroy before the share replay (same for rest of PR)